packages/cms-cli/commands/create.js

```diff
@@ -68,7 +68,11 @@ const createTemplate = (name, dest, type = 'template') => {
 function configureCreateCommand(program) {
   program
     .version(version)
-    .description('Create HubSpot CMS assets')
+    .description(
+      `Create HubSpot CMS assets. Supported assets are ${Object.values(
+        TYPES
+      ).join(', ')}.`
```

Comment:
Should there be an and after the last comma?

Reply:
Fixed in https://github.com/HubSpot/hubspot-cms-tools/pull/183